src/main/java/gmap_distance_matrix/Main.java

```diff
@@ -1,7 +1,140 @@
 package gmap_distance_matrix;
 
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.reflect.TypeToken;
+import okhttp3.HttpUrl;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.lang.reflect.Type;
+import java.util.*;
+
 public class Main {
-    public static void main(String[] args) {
-        System.out.println("Hello world!");
+    public static final String Data_FILE_PATH = "input/input.json";
+
+    public static void main(String[] args) throws IOException {
+        //get data from user
```

Comment:
I think, we shoud start using external args as inputs since these inputs then, can also be provided by other processes as well as humans. Its better to keep the application non-interactive.